geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/CreateMappingCommand.java

```diff
@@ -78,7 +78,8 @@ public ResultModel createMapping(
   }
 
   @Override
-  public void updateClusterConfig(String group, CacheConfig cacheConfig, Object element) {
+  public boolean updateClusterConfig(String group, CacheConfig cacheConfig, Object element,
+      ResultModel resultModel) {
```

Comment:
is this resultModel used in any implementation?

Reply:
Not yet. We wanted to due the refactoring of `updateClusterConfig` as an initial PR. Individual commands will use this as they are refactored to use `SingleGfshCommand`. Initial work on refactoring `AlterAsyncEventQueue` is what prompted this change.